uint8       Object_ID
float32     DistLong
float32     DistLat
float32     VrelLong
uint8       DynProp
float32     VrelLat
float32     RCS